src/init/mod.rs

```diff
@@ -109,6 +109,10 @@ fi"#,
             );
             Some(script)
         }
+        Some("ion") => {
+            let script = format!("eval $({} init ion --print-full-init)", starship);
+            Some(script)
+        }
```

Comment:
Can't comment on the relevant part, but maybe change L.129: "For the time being, we support bash, zsh and fish."?

Reply:
I added a reference to `ion` on the line you mentioned and also to the install.sh script. I think this should be all of it (ignoring the documentation in other languages).